workflows/kneaddata_paired.smk

```diff
@@ -14,17 +14,27 @@ rule kneaddata:
         fwd = os.path.join(input_folder, "{sample}_1.fastq.gz"),
         rev = os.path.join(input_folder, "{sample}_2.fastq.gz"),
     output:
-        fwd = os.path.join(kneadfolder, "{sample}_kneaddata_paired_1.fastq"),
-        rev = os.path.join(kneadfolder, "{sample}_kneaddata_paired_2.fastq"),
+        fwd = temp(os.path.join(kneadfolder, "{sample}_kneaddata_paired_1.fastq")),
+        rev = temp(os.path.join(kneadfolder, "{sample}_kneaddata_paired_2.fastq")),
         log = os.path.join(kneadfolder, "{sample}_kneaddata.log")
     run:
         shell("kneaddata --input {{input.fwd}} --input {{input.rev}} --reference-db /hg37 --output {} --output-prefix {{wildcards.sample}}_kneaddata --trimmomatic /opt/conda/share/trimmomatic".format(kneadfolder))
 
+rule compressdata:
+    input: 
+        fwd= os.path.join(kneadfolder, "{sample}_kneaddata_paired_1.fastq"),
+        rev= os.path.join(kneadfolder, "{sample}_kneaddata_paired_2.fastq")
+    output:
+        fwd= os.path.join(kneadfolder, "{sample}_kneaddata_paired_1.fastq.gz"),
+        rev= os.path.join(kneadfolder, "{sample}_kneaddata_paired_2.fastq.gz")
```

Comment:
These need to be the input for some rule (namely `kneaddata_report`, instad of the `.fastq` versions). If they're not needed as inputs for anything, this rule won't get run

Reply:
> These need to be the input for some rule (namely `kneaddata_report`, instad of the `.fastq` versions). If they're not needed as inputs for anything, this rule won't get run

Oh, I completely forgot about this element, that was something I wrote down in the last meeting notes but totally forgot to implement. I'll fix that now!